[DTM] - Documentation changes for CD 1020.1.10 (#2201)
SoftwareAG/c8y-docs: content/dtm/asset-types-bundle/asset-properties.md

#### Complex asset property {#complex-asset-property}

> A complex asset property consists of multiple key-value pairs, meaning, it contains a root object followed by one or more key-value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key-value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. You can expand each key-value pair to see a specific property. When you add a new key-value pair, it automatically expands.

Reviewer: Add a line to explain how user can collapse or expand a row to hide or view the property details

Author: Added.

> A complex asset property consists of multiple key-value pairs, meaning, it contains a root object followed by one or more key-value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key-value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. On the right side of each key-value pair, an expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i> is available. On click of expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i> you can see a specific property. On click of collapse icon <i class="dlt-c8y-icon-collapse-arrow text-muted icon-20"></i> you can hide a specific property. When you add a new key-value pair, it automatically expands.

Reviewer 2: ```suggestion
A complex asset property consists of multiple key-value pairs, meaning, it contains a root object followed by one or more key-value pairs. You can select the complex asset property option by switching on the toggle. If you switch on the toggle, you must define each key-value pair as well as providing a type for each pair. You can arrange the properties in the required order by dragging and dropping them. Hover over the row to see the drag icon. Click and drag the drag icon for reordering. On the right side of each key-value pair, you see an expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i>. Click the expand icon <i class="dlt-c8y-icon-expand-arrow text-muted icon-20"></i> to see a specific property. Note that when you add a new key-value pair, it automatically expands. To hide it, click the collapse icon <i class="dlt-c8y-icon-collapse-arrow text-muted icon-20"></i>.
```
